editor/inspector/editor_inspector.cpp

```diff
@@ -1757,8 +1759,132 @@ Size2 EditorInspectorCategory::get_minimum_size() const {
 	return ms;
 }
 
+void EditorInspectorCategory::_collect_properties(const Object *p_object, LocalVector<String> &r_properties) const {
+	List<PropertyInfo> property_list;
+	p_object->get_property_list(&property_list, true);
+
+	String current_category;
+	for (const PropertyInfo &prop_info : property_list) {
+		if (prop_info.usage & PROPERTY_USAGE_GROUP) {
+			continue;
+		}
+		if (prop_info.usage & PROPERTY_USAGE_CATEGORY) {
+			current_category = prop_info.name;
+			continue;
+		}
+		if (!(prop_info.usage & PROPERTY_USAGE_EDITOR)) {
+			continue;
+		}
+		if (!current_category.ends_with(info.name)) {
+			continue;
+		}
+
+		r_properties.push_back(prop_info.name);
+	}
+}
+
 void EditorInspectorCategory::_handle_menu_option(int p_option) {
 	switch (p_option) {
+		case MENU_COPY_VALUE: {
+			const Object *object = EditorInterface::get_singleton()->get_inspector()->get_edited_object();
+			String category_name = info.name;
+			if (!EditorNode::get_editor_data().is_type_recognized(info.name) && ResourceLoader::exists(info.hint_string, "Script")) {
+				Ref<Script> scr = ResourceLoader::load(info.hint_string, "Script");
+				if (scr.is_valid()) {
+					category_name = scr->get_doc_class_name();
+				}
+			}
+			Dictionary clipboard;
+			LocalVector<String> properties;
+			_collect_properties(object, properties);
+
+			clipboard["path"] = category_name;
```

Comment:
`path` is a valid property name and may cause a conflict. Use some special identifier, like `@path`.